src/rust/ide/src/model/project.rs

```diff
@@ -67,6 +68,21 @@ pub trait API:Debug {
     (&self, path:&model::module::Path) -> crate::model::module::QualifiedName {
         path.qualified_module_name(self.name().deref())
     }
+
+    /// Get qualified name of the project's `Main` module.
+    ///
+    /// This module is special, as it needs to be referred by the project name itself.
+    fn main_module(&self) -> FallibleResult<model::module::QualifiedName> {
+        let main = std::iter::once(crate::ide::INITIAL_MODULE_NAME);
+        model::module::QualifiedName::from_segments(self.name(),main)
+
+        // TODO [mwu] The code below likely should be preferred but does not work:
+        //            See: https://github.com/enso-org/enso/issues/1543
```

Comment:
I think we should always explain in the comment WHY it doesn' not work - using short synopsis and use link to issue for full description. Seeing in code commented code with info "doesn't work" and a link is a very hard to follow and understand. Please describe it in such a. way that it would be understandable and the link will be just "additional info"